sound/soc/sof/intel/hda.h

```diff
@@ -742,6 +742,18 @@ static inline int hda_codec_i915_exit(struct snd_sof_dev *sdev) { return 0; }
 
 #endif
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
perhaps we can introduce a new header file as well @plbossart?

Reply:
not following the suggestion @ranj063 are you saying hda.h is becoming the dumping ground for everything and we might want to split in smaller headers?

Reply:
no no. I'm only suggesting a new header for hda-ml

Reply:
ok, let's see what we can do.